### ISSUE TITLE
Is there a way to print response to console?

### ISSUE THREAD

**Issue by 3rdey3** [vicampo/riposte#11]
I am fairly new to Racket and Riposte. I was wondering if there is a way to print the whole response to console, or print variable to console.

**Comment by jessealama** [comment]
Yes! The expression `!` evaluates to the body of the last response, and the `echo` command echoes stuff to the console. Examples:

```
GET /foo/bar responds with 2XX

echo # short form, without !, just means `echo !`

echo ! # same thing

$x := /a/b

echo $a # shows the value of $a on the console
```

Does that help?

**Comment by 3rdey3** [comment]
Yes, it is exactly what I was looking for. Thanks a lot.

It would be helpful to new comers if this feature is mentioned in the [riposte documentation](https://docs.racket-lang.org/riposte)

**Comment by jessealama** [comment]
Good idea. The documentation does need some work...
